LogMonitor/src/LogMonitor/EventMonitor.cpp

```diff
@@ -552,20 +554,19 @@ EventMonitor::PrintEvent(
 
             if (status == ERROR_SUCCESS)
             {
-                // supporting JSON fmt by default
-                auto logFmt = L"{\"Source\": \"EventLog\",\"LogEntry\": {\"Time\": \"%s\",\"Channel\": \"%s\",\"Level\": \"%s\",\"EventId\": %u,\"Message\": \"%s\"}}";;
-
-                // sanitize message
-                std::wstring msg(m_eventMessageBuffer.begin(), m_eventMessageBuffer.end());
-                Utility::SanitizeJson(msg);
-
-                std::wstring formattedEvent = Utility::FormatString(
-                    logFmt,
-                    Utility::FileTimeToString(fileTimeCreated).c_str(),
-                    channelName.c_str(),
-                    c_LevelToString[static_cast<UINT8>(level)].c_str(),
-                    eventId,
-                    msg.c_str());
+                source = L"EventLog";

```

Comment:
nit: actually, come to think of it, maybe we should take the same `struct` approach and encapsulate all these fields in a struct? Could you take on that maybe after the merge, coz will just be a refactor.

Reply:
This will go into the next PR after merge as agreed.